package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "flow-router",
+  "description": "Carefully Designed Client Side Router for Meteor.",
+  "scripts": {
+    "test": "spacejam test-packages ./ --port 9001",
```

Comment:
Mark this as `testonly`.
Add a new script with `test` which does both `lint` and `testonly`